identity-apps-core/apps/authentication-portal/src/main/webapp/self-registration.jsp

```diff
@@ -265,6 +265,22 @@
                     }
                 }, [ error, flowData && flowData.data && flowData.data.additionalData && flowData.data.additionalData.error ]);
 
+                const handleInternalPrompt = (flowData) => {
+                    let providedInputs = {};
+                    flowData.data.requiredParams.map((param) => {
+                        if (param === "origin") {
+                            providedInputs[param] = window.location.origin;
+                        }
+                    });
+                    
+                    setPostBody({
+                        flowId: flowData.flowId,
+                        actionId: "",
+                        flowType: "REGISTRATION",
```

Comment:
we don't need to send this field in the response, if we already have a flowId right?